src/Schema/Conveyor/CompositedSchemaConveyor.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Cycle\Schema\Conveyor;
+
+use Cycle\Annotated\Embeddings;
+use Cycle\Annotated\Entities;
+use Cycle\Annotated\MergeColumns;
+use Cycle\Annotated\MergeIndexes;
+use JetBrains\PhpStorm\ExpectedValues;
+use Spiral\Attributes\AnnotationReader;
+use Spiral\Attributes\AttributeReader;
+use Spiral\Attributes\Composite\SelectiveReader;
+use Spiral\Attributes\ReaderInterface;
+use Spiral\Tokenizer\ClassLocator;
+use Symfony\Component\Finder\Finder;
+use Yiisoft\Aliases\Aliases;
+use Yiisoft\Yii\Cycle\Exception\EmptyEntityPathsException;
+
+class CompositedSchemaConveyor extends SchemaConveyor
```

Comment:
Suggestion: refactoring `SchemaConveyor` to abstract `AbstractSchemaConveyor` (and move here code from `CompositeSchemaConveyor`) with abstract method `getMetadataReader()`. Classes `AttributedSchemaConveyor`, `AnnotatedSchemaConveyor` and `CompositedSchemaConveyor` will inherit `AbstractSchemaConveyor`.

Reply:
It is not necessary but possible. I'l do it